circular_buffer.py

```diff
@@ -0,0 +1,92 @@
+""" Exercise circular buffer from exercism.
+https://exercism.org/tracks/python/exercises/circular-buffer"""
+
+
+class BufferFullException(BufferError):
+    """Exception raised when CircularBuffer is full.
+
+    message: explanation of the error.
+    """
+
+    def __init__(self, message):
+        # Called when an attempt is made to add data to a full buffer
+        pass
+
+
+class BufferEmptyException(BufferError):
+    """Exception raised when CircularBuffer is empty.
+
+    message: explanation of the error.
+    """
+
+    def __init__(self, message):
+        # Called when an attempt is made to retrieve data from an empty buffer
+        pass
+
+
+class CircularBuffer:
+    """A simple circular buffer implementation."""
+
+    def __init__(self, capacity):
+        """Initialize the circular buffer with a fixed capacity.
+
+        Parameters:
+        --------
+        capacity: int
+        Maximum size of the buffer.
+        """
+
+        self.capacity = capacity  # Maximum number of elements in the buffer.
+        self.buffer = []  # Internal list for the buffer elements.
+
+    def read(self):
+        """Remove and return the oldest element from the buffer.
+
+        Raises:
+        BufferEmptyException: If the buffer is empty.
+
+        Returns:
+        self.buffer.pop(0)
```

Comment:
Write type of variable and if available, name of variable to be returned

```suggestion
        Any
```